builds/jazz-build-module/config-loader.groovy

```diff
@@ -11,24 +11,77 @@ import groovy.transform.Field
 
 echo "the module, 'config-loader', loaded successfully... congratulations..."
 
-def initialize(installerVars){
-  if(installerVars != null){
-		def resultJson = parseJson(installerVars)
+/**
+ * Load the service metadata from Catalog
+ *
+ */
+def loadConfigData(aws_credential_id, region, instance_prefix) {
+  def table_name = "${instance_prefix}_JazzConfig"
+  try {
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+		credentialsId: aws_credential_id , secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+      def config_object = sh (
+        script: "aws --region ${region} dynamodb scan --table-name $table_name  --output json" ,
```

Comment:
is it possible to use a profile?

Reply:
We have to load the utility module over here to generate the id for profile. So leaving this part

Reply:
Done